### PR TITLE
Docs: Updated README

### LINKED ISSUE #25: Update README
### Description
This proposal suggests updating the README file of the project to improve clarity, remove redundant information, and add guidance for potential contributors.

### Motivation
The current README contains redundant examples that are already documented publicly. Removing these examples will streamline the README and avoid duplication of information. Additionally, adding information on how to contribute will encourage community involvement and make it easier for new contributors to get involved with the project.

### Current Behavior
The README file includes examples demonstrating the usage of the project. However, these examples are also available in the public documentation, leading to redundancy.

### Proposed Behavior
1. Remove redundant examples from the README that are already available in the public documentation.
2. Add a section detailing how to contribute to the project, including guidelines for submitting bug reports, feature requests, and pull requests.
3. Add a section on how to build the project for development.
4. Add a section on how to build the documentation and where is the html built.

### Possible Implementation
1. Review the README and the [public documentation](https://paradigmadigital.github.io/promptmeteo-docs/) to identify redundant examples that can be removed.
2. Create a new section or subsection in the README dedicated to contributing guidelines.
3. Provide clear instructions on how to report bugs, request features, and submit pull requests.
4. Provide clear instructions on how to build the project.
5. Include information on coding standards, code review processes, and other relevant guidelines for contributors.

### PR DESCRIPTION
**Pull request by chucheria**
README has been updated with developer information and all instructions about working with the library has been moved to the public documentation.

Minor changes to the public docs to add template example and the blog post about Promptmeteo.

Closes #25

### CODE REVIEW

**Comment by chucheria** [comment]
No strict guidelines has been added for developers since we don't have a policy yet on new contributors. Every contribution is welcomed but is it up to the current contributors to accept the new development and be judges of any interaction with new contributors.

File: README.md [hunk 1]
```diff
@@ -24,257 +21,45 @@ Firstly, prompts typically encapsulate application logic in their definition, ye
 Secondly, crafting concrete prompts for each task is not only a laborious task but also a complex one. Minor alterations in the input prompt can result in different outcomes, rendering them highly error-prone. Additionally, when composing prompts, considerations extend beyond the task itself to include factors such as the specific LLM being used, the model's capacity, and other relevant aspects.
 
 
+# ⚡ Use
 
+Find all tutorials, a quick start and much more information in [our official documentation](https://paradigmadigital.github.io/promptmeteo-docs/>).
```
Comment (nimbusaeta) [inline README.md]: Link fixed
```suggestion
Find all tutorials, a quick start and much more information in [our official documentation](https://paradigmadigital.github.io/promptmeteo-docs/).
```

File: README.md [hunk 2]
```diff
@@ -24,257 +21,45 @@ Firstly, prompts typically encapsulate application logic in their definition, ye
...
-```
-
+# Contribute
 
-### ✨ Load a trained task
+We encourage you:
 
-... and loaded again to make new predictions.
+* to [open issues](https://github.com/paradigmadigital/Promptmeteo/issues/)
+* to [contact us](https://github.com/paradigmadigital/Promptmeteo/graphs/contributors>) to talk about the library
```
Comment (nimbusaeta) [inline README.md]: Link fixed
```suggestion
* to [contact us](https://github.com/paradigmadigital/Promptmeteo/graphs/contributors) to talk about the library
```

File: README.md [hunk 3]
```diff
@@ -24,257 +21,45 @@ Firstly, prompts typically encapsulate application logic in their definition, ye
...
+* to [develop open issues or fix bugs](https://github.com/paradigmadigital/Promptmeteo/issues?q=is%3Aissue+is%3Aopen+>)
```
Comment (nimbusaeta) [inline README.md]: Link fixed
```suggestion
* to [develop open issues or fix bugs](https://github.com/paradigmadigital/Promptmeteo/issues?q=is%3Aissue+is%3Aopen+)
```

File: README.md [hunk 4]
```diff
@@ -24,257 +21,45 @@ Firstly, prompts typically encapsulate application logic in their definition, ye
...
-clf = DocumentClassifier(
-        language            = 'en',
-        model_provider_name = 'hf_pipeline',
-        model_name          = 'google/flan-t5-small',
-    ).load_model("hello_world.meteo")
+## Develop
 
-clf.predict(['so cool!!'])
-```
+Clone the repository and create a new branch based on [semantic messages](https://www.conventionalcommits.org/en/v1.0.0/#summary>) (_feat/feature_, _chore_, etc):
```
Comment (nimbusaeta) [inline README.md]: Link fixed
```suggestion
Clone the repository and create a new branch based on [semantic messages](https://www.conventionalcommits.org/en/v1.0.0/#summary) (_feat/feature_, _chore_, etc):
```

**Comment by nimbusaeta** [comment]
Looks great! I only fixed some links.

**Pull request merged**